source/priority_queue.c

```diff
@@ -176,8 +176,6 @@ void aws_priority_queue_init_static(
     AWS_ZERO_STRUCT(queue->backpointers);
 
     aws_array_list_init_static(&queue->container, heap, item_count, item_size);
-
-    AWS_POSTCONDITION(aws_priority_queue_is_valid(queue));
```

Comment:
Why remove this?

Reply:
i didn't mean to remove this, thanks for the catch.